.idea/Clio_wallet.iml

```diff
@@ -39,37 +39,66 @@
     <orderEntry type="library" scope="PROVIDED" name="activerecord (v7.0.3, rbenv: 3.0.4) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="activestorage (v7.0.3, rbenv: 3.0.4) [gem]" level="application" />
     <orderEntry type="library" scope="PROVIDED" name="activesupport (v7.0.3, rbenv: 3.0.4) [gem]" level="application" />
+    <orderEntry type="library" scope="PROVIDED" name="addressable (v2.8.0, rbenv: 3.0.4) [gem]" level="application" />
```

Comment:
We should delete this file as it is editor specific, and add it in .[gitignore](https://git-scm.com/docs/gitignore), but can do it separately so no worries👍